docs/dev/release-checklist.md

```diff
@@ -0,0 +1,15 @@
+# Release Checklist
+
+- [ ] Security fixes (make sure all available undisclosed security fixes are merged)
+- [ ] Bump the version
+- [ ] Make sure `yarn.lock` is up to date
+- [ ] Make sure translations are up to date
+- [ ] Write release notes
+  - [ ] General description
+  - [ ] Things requiring special action beside updating the software
+  - [ ] New features
+  - [ ] Bug fixes
+  - [ ] Add all contributors
+- [ ] Run the tests described in the release test document (internal)
```

Comment:
Do we just want to include these tests here?

Reply:
Yes, I think it's a good idea to include a more specific description of what to test.